plugins/mac-capture/data/locale/en-US.ini

```diff
@@ -37,3 +37,5 @@ SCK.AudioUnavailable="Audio capture requires macOS 13 or newer."
 SCK.CaptureTypeUnavailable="Selected capture type requires macOS 13 or newer."
 SCK.Method="Method"
 SCK.Restart="Restart capture"
+Warning.Effect.Audio.VoiceIsolation="The macOS system effect 'Voice Isolation' is active on the selected device"
+Warning.Effect.Audio.WideSpectrum="The macOS system effect 'Wide Spectrum' is active on the selected device"
```

Comment:
The message is incorrect, because the settings are active for the application, not the device. There is no way to disable it for one device and have it enabled for another.

Saying "on the selected device" would imply that I can switch to another device and it could have a different state, but that ain't so - voice isolation (the most common setting that might be encountered) will be enabled for all input devices.

Reply:
I disagree, this setting is "per application" for a reason - namely that the vast majority of consumer applications is not a compositing/editing application but single-input voice conference or audio/video call applications, and the system-wide voice isolation feature was designed with that use case in mind.

We should not communicate something to our users that is factually wrong (it is *not* enabled for the device, it is enabled for the application), because then it is us who confuse the users ("but OBS told me it was enabled for the device, where I can disable it for the device?").

The warning should simply state "Voice Isolation is enabled for OBS Studio. Disable if unfiltered audio input is desired".

No opinions. No value statements. Just the facts.